vm/src/vm.rs

```diff
@@ -987,6 +987,53 @@ impl VirtualMachine {
         }
     }
 
+    pub fn map_elements<F, T, R>(&self, obj: &PyObjectRef, mut f: F) -> PyResult<PyResult<Vec<R>>>
+    where
+        T: TryFromObject,
+        F: FnMut(T) -> PyResult<R>,
+    {
+        self.map_iterable_object(obj, |elem| f(T::try_from_object(self, elem)?))
+    }
```

Comment:
This function not been use yet, maybe remove it?

Reply:
Generally, yes. having unused code is a sort of cost. Do you have plan to use it in other patch?